docker-compose.yml

```diff
@@ -12,6 +12,18 @@ services:
     ports:
       - 5000:5000
 
+  tweet-api:
```

Comment:
Would it make sense to just call it `api`? Simpler and faster to type; and what we're doing isn't tied to Twitter, is it?

Reply:
@danasaur feel free to name yours `api` and rename mine to, say, `mock-api` or `fake-api` or something